plugins/modules/system/xfconf.py

```diff
@@ -16,6 +16,9 @@
 description:
   - This module allows for the manipulation of Xfce 4 Configuration via
     xfconf-query.  Please see the xfconf-query(1) man pages for more details.
+    
+    TO-DO: add unit tests, add a 'list' state to retrieve properties, add recursive reset
+version_added: "2.8"
```

Comment:
Please remove this line.

Also don't put TODOs in the documentation.

Reply:
fixing